Api/Connection.php

```diff
@@ -15,7 +15,7 @@ public function post($url, $bodyArray = false)
             $encodedBody = $bodyArray ? json_encode($bodyArray) : '';
             $parsedUrl = parse_url($url);
             $headers = [
-                'Content-Type: text/plain',
+                'Content-Type: application/json',
```

Comment:
I don't really remember why we changed the content-type of the headers, but the activity api call is being rejected if it's not application/json type.

Reply:
are tracking calls working if the content type is JSON?

Reply:
Nope, response code is 401.

Reply:
ok, so how do we fix this?

Reply:
I am not really sure how to proceed here, because sending curl call requires the headers content-type of json. A solution to that can be passing additional parameter to Api/Connection post method where we can place conditional that is replacing content-type based on call needs.